e2e_test/nexmark/create_sources.slt.part

```diff
@@ -36,7 +36,7 @@ CREATE SOURCE nexmark (
 ) WITH (
     connector = 'nexmark',
     nexmark.split.num = '2',
-    nexmark.min.event.gap.in.ns = '100'
+    nexmark.min.event.gap.in.ns = '100000000'
```

Comment:
Is this change is to avoid too much metrics when debugging the metrics service with a nexmark source?

Reply:
Pushed mistakenly. 🤣 Will revert.